libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/ElementThemeApp.kt

```diff
@@ -39,6 +44,15 @@ fun ElementThemeApp(
         appPreferencesStore.getThemeFlow().mapToTheme()
     }
         .collectAsState(initial = Theme.System)
+    LaunchedEffect(theme) {
+        AppCompatDelegate.setDefaultNightMode(
+            when (theme) {
+                Theme.System -> AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM
+                Theme.Light -> AppCompatDelegate.MODE_NIGHT_NO
+                Theme.Dark -> AppCompatDelegate.MODE_NIGHT_YES
+            }
+        )
+    }
```

Comment:
Note that it could be done elsewhere, for instance in `RootPresenter`, but when the value is the same, `setDefaultNightMode` has no effect, so I guess it's fine to do it here.